### PR TITLE
IC-1057 - Session progress table

### PR DESCRIPTION
**Pull request by Gweaton**
## What does this pull request do?

**Jira card URL:**
https://dsdmoj.atlassian.net/browse/IC-1052

## Changes in this PR:
Added in the table and tag component, and styled it as per design

## Screenshots of UI changes:
Screenshot 2021-02-08 at 12 00 01

### CODE REVIEW

File: assets/sass/components/_badges-tags.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+a.govuk-link {
+  font-weight: 400;
```
Comment (lawrence-forooghian) [inline assets/sass/components/_badges-tags.scss:2]: Perhaps I'm missing something, but does this change the weight of all of the links in the app? Is that the intention?

Comment (isratc) [reply assets/sass/components/_badges-tags.scss]: Yes it was meant to be wrapped around the table tag we had a bit of a git mashup yesterday forgot to add in

File: assets/sass/components/_badges-tags.scss [hunk 2]
```diff
@@ -1,8 +1,7 @@
-a.govuk-link {
-  font-weight: 400;
-}
-
 .govuk-table {
+  a.govuk-link {
+    font-weight: 400;
```
Comment (Gweaton) [inline assets/sass/components/_badges-tags.scss:3]: Hey, in your other PR I saw you put: `font-weight: normal; ` - I didn't know it was a thing, but I think it's nicer than using the magic number of `400` - do you think you'd adding it here too? It's also nice to have consistency across the styles.

Comment (isratc) [reply assets/sass/components/_badges-tags.scss:3]: sure, but it's truly not a big deal

**Review by isratc — APPROVED** [review]

**Pull request merged**